streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -473,38 +496,34 @@ public void run() {
     }
 
     /**
-     * Produces a string representation containing useful information about Kafka Streams
-     * Such as thread IDs, task IDs and a representation of the topology. This is useful
-     * in debugging scenarios.
+     * Produce a string representation containing useful information about this {@code KafkaStream} instance such as
+     * thread IDs, task IDs, and a representation of the topology DAG including {@link StateStore}s (cf.
+     * {@link TopologyBuilder} and {@link KStreamBuilder}).
+     *
      * @return A string representation of the Kafka Streams instance.
      */
     @Override
     public String toString() {
-        return toString("");
-    }
-
-    /**
-     * Produces a string representation containing useful information about Kafka Streams
-     * such as thread IDs, task IDs and a representation of the topology starting with the given indent. This is useful
-     * in debugging scenarios.
-     * @return A string representation of the Kafka Streams instance.
-     */
-    public String toString(final String indent) {
-        final StringBuilder sb = new StringBuilder(indent + "KafkaStreams processID:" + processId + "\n");
+        final StringBuilder sb = new StringBuilder("\tKafkaStreams processID:" + processId + "\n");
         for (final StreamThread thread : threads) {
-            sb.append(thread.toString(indent + "\t"));
+            sb.append(thread.toString("\t\t"));
```

Comment:
If you do want to remove `toString(indent)` function, then here it should be a single `\t`. And ditto for line 507 above where indent should be `""`. BTW I think it is OK to keep this function as is for users to choose their own wrapping comments of string block.